new/t1.rb

```diff
@@ -0,0 +1,11 @@
+class Plumbus
+private
```

Comment:
Indent access modifiers like `private`.

```suggestion
  private
```